### PR TITLE
[#11503] Suggesting edits to the developer guidelines

### LINKED ISSUE #11503: Logging into an instance development guidelines clarification
Hi,

I have been working through [developer guidelines](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#logging-in-to-a-teammates-instance) to set up and run teammates' development environment and had some confusion when trying to log into an instance. Specifically, in the section "[Logging in to a TEAMMATES instance](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#as-administrator)" as administrator it says that I will be granted access if the account has admin permission as defined in build.properties. However, to be able to log in as an administrator with my username I had to run the datastore emulator, not just edit build.properties. It took my team of students and me some time to figure that out since it wasn't intuitive to us or explicitly mentioned in the guide.

Therefore, I wanted to suggest two small changes to the developer's guide, to make this a bit more clear to student developers like myself: 
1) Add a sentence about running the datastore emulator in the first bullet point of  "Logging in to a TEAMMATES instance" as administrator. 
2) Switch around sections about[ logging into an instance](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#logging-in-to-a-teammates-instance) and [running the datastore emulator](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#as-administrator) so that they appear in chronological order. 

Please let me know if I misunderstood something and thank you!

### PR DESCRIPTION
**Pull request by egorovak**
Fixes #11503

Suggesting some edits to the developer guidelines s described in the related issue

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -109,7 +152,7 @@ This instruction set applies for both dev server and production server, with sli
 ### As administrator
 
 1. Go to any administrator page, e.g `/web/admin/home`. You may be prompted to log in.
-   You will be granted access only if your account has admin permission as defined in `build.properties`.
+   You will be granted access only if your account has admin permission as defined in `build.properties`. To log in using your admin name you have to edit `build.properties` to grant your username permission and run datastore emulator as defined above.
```
Comment (moziliar) [inline docs/development.md:155]: If we are to make it more straightforward, some example might work even better:)

Comment (wkurniawan07) [reply docs/development.md:155]: This is not the right place to do it:
- As mentioned in the issue thread, it is not the logging in action itself that requires Datastore/emulator.
- The need for Datastore/emulator applies whether you want to be recognized (notice that I didn't say "log in") as student or instructor or admin.

**Comment by daongochieu2810** [comment]
@egorovak Let us know if you have resolve the comments made by others

**Comment by madanalogy** [comment]
Closing due to inactivity.

**Pull request closed without merging**